mmdet/models/detectors/base.py

```diff
@@ -198,6 +198,13 @@ def _parse_losses(self, losses):
         loss = sum(_value for _key, _value in log_vars.items()
                    if 'loss' in _key)
 
+        # If the loss_vars has different length, GPUs will wait infinitely
+        if dist.is_available() and dist.is_initialized():
```

Comment:
You can directly use `reduce_mean` in mmdet/core/utils/dist_utils.py

Reply:
Both ways are good

Reply:
It's better to be able to write uniformly.

Reply:
The communication in the [following lines](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/detectors/base.py#L206) are also through `torch.dist`. So I just use a similar way to do this. If you want to use the utilities from mmdet, I suggest a more complete style check.

Reply:
Okay